test/cmdlineTests.sh

```diff
@@ -337,7 +337,6 @@ printTask "Testing assemble, yul, strict-assembly and optimize..."
     # Non-empty code results in non-empty binary representation with optimizations turned off,
     # while it results in empty binary representation with optimizations turned on.
     test_solc_assembly_output "{ let x:u256 := 0:u256 }" "{ let x:u256 := 0:u256 }" "--yul"
-    test_solc_assembly_output "{ let x:u256 := 0:u256 }" "{ }" "--yul --optimize"
```

Comment:
Could add a test that checks that it fails.

Reply:
done.